### PR TITLE
dotCMS/core#16274 Add tests to dot-label and update utils tests

### PR DESCRIPTION
**Pull request by fmontes**

### CODE REVIEW

File: projects/dotcms-field-elements/src/utils/utils.spec.tsx [hunk 1]
```diff
@@ -1,68 +1,107 @@
...
+    getStringFromDotKeyArray,
     getTagError,
-    getErrorClass
+    getTagHint,
+    updateStatus
 } from './utils';
 
+describe('getClassNames', () => {
+    it('should returns field CSS classes', () => {
+        const status = { dotValid: false, dotTouched: false, dotPristine: true };
```
Comment (freddyucv) [inline projects/dotcms-field-elements/src/utils/utils.spec.tsx]: I think one more with { dotValid: true, dotTouched: true, dotPristine: false };

Comment (fmontes) [reply projects/dotcms-field-elements/src/utils/utils.spec.tsx]: Done

File: projects/dotcms-field-elements/src/components/dot-label/dot-label.e2e.ts [hunk 2]
```diff
@@ -1,26 +1,101 @@
...
+
+    describe('<slot />', () => {
+        beforeEach(async () => {
+            page = await newE2EPage({
+                html: `
+                <dot-label labe="hello world">
+                    <h1>into the slot</h1>
+                </dot-label>`
+            });
+            element = await page.find('dot-label');
```
Comment (alfredo-dotcms) [inline projects/dotcms-field-elements/src/components/dot-label/dot-label.e2e.ts]: use the function getLabel

File: projects/dotcms-field-elements/src/components/dot-label/dot-label.e2e.ts [hunk 3]
```diff
@@ -1,26 +1,101 @@
...
+
+            it('should render empty string with no value', async () => {
+                element.setProperty('label', undefined);
+                await page.waitForChanges();
+                const text = await getText();
+                expect(text.innerText).toBe('');
+            });
+        });
+
+        describe('required', async () => {
```
Comment (alfredo-dotcms) [inline projects/dotcms-field-elements/src/components/dot-label/dot-label.e2e.ts]: async not needed

File: projects/dotcms-field-elements/src/utils/utils.spec.tsx [hunk 4]
```diff
@@ -1,92 +1,149 @@
...
         expect(items).toEqual([{ label: 'key1', value: 'A' }, { label: 'key2', value: 'B' }]);
     });
 });
 
-describe('getOriginalStatus', () => {
-    it('should returns initial field Status', () => {
-        const status = getOriginalStatus();
-        expect(status).toEqual({ dotValid: true, dotTouched: false, dotPristine: true });
+describe('getErrorClass', () => {
+    it('should return rrror CSS', () => {
```
Comment (alfredo-dotcms) [inline projects/dotcms-field-elements/src/utils/utils.spec.tsx]: typo

File: projects/dotcms-field-elements/src/utils/utils.spec.tsx [hunk 5]
```diff
@@ -1,92 +1,149 @@
...
+            dotValid: true,
+            dotTouched: false,
+            dotPristine: true
+        });
     });
-    it('should Not returns Hint tag', () => {
-        const hint = '';
-        const jsxTag: any = getTagHint(hint);
-        expect(jsxTag).toEqual('');
+    it('should return field Status with overwrite dotValid equal false', async () => {
```
Comment (alfredo-dotcms) [inline projects/dotcms-field-elements/src/utils/utils.spec.tsx]: async not needed

File: projects/dotcms-field-elements/src/components/dot-label/dot-label.e2e.ts (hunk 2 above)
Comment (fmontes) [reply projects/dotcms-field-elements/src/components/dot-label/dot-label.e2e.ts]: Is not the same, the `getLabel` return the `<label>` inside `<dot-label>`

**Pull request merged**